virtool/pg/utils.py

```diff
@@ -36,7 +37,8 @@ async def connect(postgres_connection_string: str) -> AsyncEngine:
     try:
         pg = create_async_engine(
             postgres_connection_string,
-            json_serializer=pretty_dumps,
+            json_serializer=pretty_orjson_serializer,
```

Comment:
Why pretty?

Reply:
I will change this to regular dumps and get rid of `pretty_orjson_serializer`.